packages/babel-plugin-transform-async-to-generator/test/fixtures/export-async/import-and-export/expected.js

```diff
@@ -3,9 +3,9 @@
 Object.defineProperty(exports, "__esModule", {
   value: true
 });
-exports.foo = undefined;
+exports.foo = foo;
 
-let foo = exports.foo = (() => {
+let foo = (() => {
```

Comment:
You should also move the declaration of the function before `exports.foo = ...`, otherwise code like this throws because of TDZ (you are using `foo` before it is defined)